src/salaryAndBenefits/SalaryAndBenefits.tsx

```diff
@@ -1,23 +1,26 @@
 import styles from "./salaryAndBenefits.module.css";
 import Text from "src/components/text/Text";
-import {SalaryAndBenefits as SalaryAndBenefitsPayload} from "studio/lib/payloads/salaryAndBenefits";
-import Benefit from './components/benefit/Benefit';
+import { SalaryAndBenefits as SalaryAndBenefitsPayload } from "studio/lib/payloads/salaryAndBenefits";
```

Comment:
until we've done a refactor let's use the structure of the repo
```suggestion
import { SalaryAndBenefitsPage } from "studio/lib/payloads/salaryAndBenefits";
```